drivers/lora/sx126x.c

```diff
@@ -443,6 +443,15 @@ void SX126xWakeup(void)
 	dev_data.mode = MODE_STDBY_RC;
 }
 
+uint32_t SX126xGetDio1PinState(void)
+{
+	if (gpio_pin_get(dev_data.dio1, GPIO_DIO1_PIN) > 0) {
```

Comment:
I'm a fan of ternary statements in situations like this:
```
return gpio_pin_get(dev_data.dio1, GPIO_DIO1_PIN) > 0 ? 1U : 0U;
```
Just a suggestion.

Reply:
Done.